src/CLI/clitree/cli-xml/ifa.xml

```diff
@@ -57,13 +95,46 @@ limitations under the License.
              optional="true">
       </PARAM>
        <ACTION builtin="clish_pyobj">sonic_cli_ifa get_tam_ifa_flow_stats ${name}</ACTION>
+       <DOCGEN>
+            <DESCRIPTION>
+                Shows IFA flow statistics.
+            </DESCRIPTION>
+            <USAGE>
+                Use this command to display packet count and byte count for matcing IFA flow.
+            </USAGE>
+            <EXAMPLE summary="This">
+                sonic# show tam int-ifa statistics
```

Comment:
I think it is better to have examples with "show tam int-ifa statistics all" and "show tam-int-ifa statistics <flowname>